test/LinalgExt/tiling.mlir

```diff
@@ -1,6 +1,5 @@
-// RUN: mlir-proto-opt -linalg-ext-tiling="tile-sizes=2,4" %s | FileCheck %s
+// RUN: mlir-proto-opt -linalg-ext-tiling="tile-sizes=10,20" -split-input-file -canonicalize -cse %s | FileCheck %s
```

Comment:
Core has good hygiene here in that the base patterns do not involve multiple passes.
Let's please avoid this here too.

Reply:
Done.